dcat/index.html

```diff
@@ -225,31 +224,23 @@ <h3>DCAT scope</h3>
         <li>
           <a href="#Class:Resource"><code>dcat:Resource</code></a> represents an individual item in a catalog.
           This class is not intended to be used directly, but is the parent class of <a href="#Class:Dataset"><code>dcat:Dataset</code></a>, <a href="#Class:Data_Service"><code>dcat:DataService</code></a> and <a href="#Class:Catalog"><code>dcat:Catalog</code></a>.
-          Member items in a catalog should be members of one of the sub-classes, or of a sub-class of these, or of a sub-class of <a href="#Class:Resource"><code>dcat:Resource</code></a> defined in a DCAT application profile or other DCAT application.
+          Member items in a catalog should be members of one of these sub-classes, or of a sub-class of these, or of a sub-class of <a href="#Class:Resource"><code>dcat:Resource</code></a> defined in a DCAT application profile or other DCAT application.
           <a href="#Class:Resource"><code>dcat:Resource</code></a> is effectively an extension point for defining a catalog of any kind of resource.
         </li>
         <li>
           <a href="#Class:Dataset"><code>dcat:Dataset</code></a> represents a dataset in a catalog.
           A dataset is a collection of data, published or curated by a single agent.
-          Data comes in many forms including numbers, words, pixels, imagery, sound and other multi-media, and potentially other types, any of which might be collected into a dataset.
```

Comment:
Why remove this?

Reply:
@dr-shorthair I re-worked the preceding section about scope to include this point (see line 216) so that the definition of dcat:Dataset is cleaner